projects/components/src/table/controls/table-controls.component.ts

```diff
@@ -238,13 +238,21 @@ export class TableControlsComponent implements OnChanges {
   }
 
   public onMultiSelectChange(selectControl: TableSelectControl, selections: TableSelectControlOption[]): void {
+    this.applySelections(selectControl, selections);
+
     this.selectChange.emit({
       select: selectControl,
       values: selections
     });
     this.diffSelections();
   }
 
+  private applySelections(selectControl: TableSelectControl, selections: TableSelectControlOption[]): void {
```

Comment:
Do we need to update any existing code?

Reply:
No need at the moment. This shouldn't hurt anything.